DuggaSys/gitfetchService.php

```diff
@@ -267,7 +233,6 @@ function bfs($url, $cid, $opt)
                                 }
                             }
                         }
-                        // Checks if the fetched item is of type 'dir'
```

Comment:
Removing comments makes it harder to understand the code. If the comments are not accurate, make new ones.